### DISCUSSION TITLE
Issue accessing OpenAI Whisper large model on Playground

### DISCUSSION THREAD

**Discussion by slim16165** [deepgram/community#946] [General help] [answered]
Hi everyone,

I tried using the large model of OpenAI Whisper via the Playground, but I encountered the error below while attempting to transcribe one of the sample English files.

```
err_code:"INSUFFICIENT_PERMISSIONS"
err_msg:"Project does not have access to the requested model."
request_id:"b58095d6-8802-4ab9-8376-39ac351cad76"
```

It seems that my project lacks the required permissions to access this specific model. I have basic access to the Playground, but I'm unsure what additional permissions or configurations are necessary to use the large model.

Has anyone experienced a similar issue or know how to resolve it? Any guidance would be appreciated!

**Comment by jjmaldonis** [comment, marked as answer]
Hi @slim16165, in this case the model name was spelled incorrectly: `whishper-large`. If you set the model name to `whisper-large` the request will succeed :tada:

That said, we highly encourage you to try our Nova 2 model for any language it supports. Nova 2 is much faster and has less pathologies than whisper large.

**Reply by slim16165** [reply]
Thank you very much! I have tested Nova 2, Whisper-medium, and Whisper-large. I often use these engines in parallel, reviewing the transcriptions with the help of Claude or ChatGPT. I use them to transcribe YouTube videos, environmental recordings, and calls, sometimes optimizing the audio with tools like Audacity, Auphonic, and Adobe Enhance Speech.

Currently, for Italian, Nova 2 often proves to be less accurate than Whisper. I would be happy to contribute to any tests if you find it useful.
